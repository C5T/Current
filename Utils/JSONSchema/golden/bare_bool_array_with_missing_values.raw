{"ObjectOrArray":{"fields":{"":{"Bool":{"values_false":1,"values_true":2,"nulls":2}}},"instances":1,"nulls":0}}