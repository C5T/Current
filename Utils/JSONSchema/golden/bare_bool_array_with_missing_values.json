[true,null,true,false,null]
